frontend/packages/dev-console/src/components/pipelines/pipelinesResources/pipelinesClusterOptions.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { TextInputTypes } from '@patternfly/react-core';
+import { InputField, CheckboxField, InputTextAreaField } from '../../formik-fields';
+
+const PipelinesClusterOptions: React.FC = () => (
+  <React.Fragment>
+    <InputField
+      type={TextInputTypes.text}
+      name="params.url"
+      label="Url"
+      helpText="Host url of the master node."
+      required
+    />
+    <InputField
+      type={TextInputTypes.text}
+      name="params.username"
+      label="Username"
+      helpText="The user with access to the cluster."
+      required
+    />
+    <InputField
+      type={TextInputTypes.password}
+      name="params.password"
+      label="Password"
+      helpText="Please provide Password."
+    />
+    <CheckboxField
+      name="params.insecure"
+      label="Insecure"
+      helpText="Indicate server should be accessed without verifying the TLS certificate"
+    />
+    <InputTextAreaField
```

Comment:
TextArea should be controlled as can be resized, it's on me

Reply:
I think this could use `DroppableFileInputField` instead of the plain text area field. That is what is used in routing for ca data and in create secret form as well. It would be more consistent with other form fields like this but depends on what UX wants here.

Reply:
Let me take a look